opendocdb-cts/internal/mongosh/fixture_test.go

```diff
@@ -141,6 +157,7 @@ func TestConvertFixtures(t *testing.T) { //nolint:revive // exceeds number of li
 			},
 			expected: `
 			db.c.insertMany([{"_id": "decimal128", "v": Decimal128("42.13")}]);`,
+			skip: "https://github.com/OpenDocDB/cts/issues/34",
```

Comment:
Why do we have a test for the current but invalid behavior, and then skip it?

Reply:
> Fixed how?
The test currently checks for the correct behavior.

> Why do we introduce test skipping in general?
Because this test fails due to our invalid handling of decimal128 values for which we have an open issue about.